config/200-role.yaml

```diff
@@ -59,3 +59,31 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "update"]
     resourceNames: ["webhook-certs"]
+---
```

Comment:
aggregating the role around being able to view clustertasks to the pre-existing k8s view role is another option around giving read access to `ClusterTasks` out of the box